data-prepper-plugins/s3-source/src/test/java/com/amazon/dataprepper/plugins/source/SqsWorkerTest.java

```diff
@@ -94,8 +140,13 @@ void processSqsMessages_should_return_number_of_messages_processed() {
         assertThat(messagesProcessed, equalTo(1));
         verify(s3Service).addS3Object(any(S3ObjectReference.class));
         verify(sqsClient).deleteMessageBatch(any(DeleteMessageBatchRequest.class));
+        verify(sqsMessagesReceivedCounter).increment(1);
+        verify(sqsMessagesDeletedCounter).increment(1);
+        assertThat(actualDelay, lessThanOrEqualTo(Duration.ofHours(1).plus(Duration.ofMinutes(5))));
```

Comment:
Five minutes is a lot of room for error. Why did you go this high rather than 1 second? This test should run quickly so the difference really should only be in milliseconds.

Reply:
My intention was to do 5 seconds, my bad.
It's taking around PT1H1.370169S which is 1.3 seconds. I believe it's more due to the logic in test itself from line 111 - 124 before calling `processSqsMessages` on line 126.